sdlc_xmalab.py

```diff
@@ -166,9 +173,8 @@ def load_project(working_dir=os.getcwd()):
         dlc_yaml = dlc_config_loader.load(dlc_config)
```

Comment:
Because of the behavior of ruamel, we can actually reduce the number of lines the with statement is open for (again, code that I wrote as a programming noob coming back to bite us, sorry!). So

```python
dlc_config_loader = YAML()
with open(project['path_config_file'], 'r') as dlc_config:
    dlc_yaml = dlc_config_loader.load(dlc_config)

# Rest of func code unindented here
...
```

Reply:
done